lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -511,4 +512,17 @@ public S3FileSystemConfig setSupportsExclusiveCreate(boolean supportsExclusiveCr
         this.supportsExclusiveCreate = supportsExclusiveCreate;
         return this;
     }
+
+    public boolean isEnableSetContentMd5()
+    {
+        return enableSetContentMd5;
+    }
+
+    @Config("s3.enable-set-content-md5")
+    @ConfigDescription("Whether to set content md5 for s3 write requests")
```

Comment:
The description should mention the _effect_ of changing this setting, so users can understand why they'd want to change this. Copying from S3 docs:
> adds end-to-end integrity check but introduces additional latency to compute the MD5 digest

Reply:
@nineinchnick thank you for the comment, updated the description to be more informative